client/post-editor/editor-gutenberg-blocks-warning-dialog/index.jsx

```diff
@@ -128,6 +138,45 @@ class EditorGutenbergBlocksWarningDialog extends Component {
 	}
 }
 
+const mapDispatchToProps = dispatch => ( {
+	switchToGutenberg: ( siteId, gutenbergUrl ) => {
+		dispatch(
+			withAnalytics(
+				composeAnalytics(
+					recordGoogleEvent(
+						'Gutenberg Opt-In',
+						'Clicked "Switch to the new editor" in the blocks warning dialog.',
+						'Opt-In',
+						true
+					),
+					recordTracksEvent( 'calypso_gutenberg_opt_in', {
+						opt_in: true,
+					} ),
+					bumpStat( 'gutenberg-opt-in', 'Calypso Dialog Opt In' )
+				),
+				setSelectedEditor( siteId, 'gutenberg', gutenbergUrl )
+			)
+		);
+	},
+	useClassic: () => {
+		dispatch(
+			withAnalytics(
+				composeAnalytics(
+					recordGoogleEvent(
+						'Gutenberg Opt-Out',
+						'Clicked "Use the classic editor" in the blocks warning dialog.',
+						'Opt-In',
+						false
+					),
+					recordTracksEvent( 'calypso_gutenberg_use_classic_editor' ),
+					bumpStat( 'selected-editor', 'calypso-gutenberg-use-classic-editor' )
```

Comment:
Ditto to not using this bump stat, as we're not really switching, we're choosing to keep using classic

Reply:
I'd be open to maybe grouping this under something else if you think it's still important to track